src/data-types-lib/index.js

```diff
@@ -44,9 +44,9 @@ export const
 
         return value === EMPTY_FIELD_VALUE ?
           uiTypes[uiType].EMPTY_VALUE :
-          converter.format(value);
+          converter.format(value, { i18n });
       },
-      parse(value) {
+      parse(value, { i18n } = {}) {
```

Comment:
Default value does not make any sense because
1. **i18n** is queried with **formatDecimalNumer()**.
2. **i18n** is required according to **src/crudeditor-lib/index.js**

Change arguments from `(value, { i18n } = {})` to `({ value, i18n })`